spec/system/assignment_spec.rb

```diff
@@ -3,18 +3,112 @@
 require "rails_helper"
 
 describe "Assignments", type: :system do
+  let(:primary_mentor) { FactoryBot.create(:user) }
+  let(:group) { FactoryBot.create(:group, primary_mentor: primary_mentor) }
+  let(:mentor) do
+    user = FactoryBot.create(:user)
+    FactoryBot.create(:group_member, group: group, user: user, mentor: true)
+    user
+  end
+  let(:member) do
+    user = FactoryBot.create(:user)
```

Comment:
We can do something like this to simplify 
```suggestion
    user = FactoryBot.create(:user, group_members:  [FactoryBot.create(:group_member, group: group)])
```